### PR TITLE
Feat/week five task solution

### PR DESCRIPTION
**Pull request by taiwoyewole**

### CODE REVIEW

File: week 5/week-five-task-solution/CAT.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        break;
+      case this.happiness >= 10:
+        console.log('Hi there. I\'m happy today!');
+        break;
+      default:
+        console.log('Hello! I\'m Kitty. It\'s nice to meet you!');
+        break;
+    }
+  }
+}
```
Comment (Easybuoy) [inline week 5/week-five-task-solution/CAT.js:71]: Good use of Switch here

**Review by Easybuoy — APPROVED** [review]
Great job with the game, it looked awesome!

File: week 5/week-five-task-solution/Roshambo/script.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    const randNum = Math.random();
+
+    if (randNum < 0.35) {
+        return 'rock';
+    } else if (randNum <= 0.67 ) {
+        return 'paper';
+    } else {
+        return 'scissors';
+    }
+}
```
Comment (Easybuoy) [inline week 5/week-five-task-solution/Roshambo/script.js:31]: You can try attempting turning this into a class

File: week 5/week-five-task-solution/CAT.js (hunk 1 above)
Comment (taiwoyewole) [reply week 5/week-five-task-solution/CAT.js:71]: Thanks

File: week 5/week-five-task-solution/Roshambo/script.js (hunk 2 above)
Comment (taiwoyewole) [reply week 5/week-five-task-solution/Roshambo/script.js:31]: Okay
I’ll try do that 

Thanks

**Pull request merged**